testsuite/MDAnalysisTests/analysis/test_psa.py

```diff
@@ -126,10 +125,12 @@ def test_get_coord_axes_bad_dims(self):
             PSA.get_coord_axes(np.zeros((5,5,5,5)))
 
     def test_dist_mat_to_vec_func_out_of_bounds(self):
-        """Test that ValueError is raised when i or j is
+        """Test that ValueError is raised when i or j or both are
         out of bounds of N"""
 
         with self.assertRaises(ValueError):
+            PSA.dist_mat_to_vec(5, 6, 4)
```

Comment:
Have each of these line in a different `with` block, or in a different test. The way it is now makes it difficult to now which one failed if there is a failure.